js/events/pointer.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
-var support = require("../core/utils/support"),
+var jQuery = require("jquery"),
```

Comment:
We should not require jquery outside the jquery integration folder. I think these changes should be moved there.

Reply:
Or maybe you need to configure `special` via registerEvent if you want to patch noJQuery strategy too.

Reply:
fixed